llvm/lib/Transforms/Utils/SimplifyCFG.cpp

```diff
@@ -1853,6 +1853,113 @@ static void hoistConditionalLoadsStores(
   }
 }
 
+static std::optional<bool>
+visitConditions(Value *V, const Value *baseCond, const BasicBlock *contextBB,
```

Comment:
Variable names should start with uppercase letter.

Reply:
done